wiki/Tournaments/3WC/2021/en.md

```diff
@@ -0,0 +1,122 @@
+    - If a country has more signups than the team size allows for, the captain will be tasked with forming the team.
+
+### Match instructions
+
+- Round of 16 and Quarterfinals
+    - Best of 11, 2 Bans, 5NM/2HD/2HR/3DT/2FM/1TB
+- Semifinals
+    - Best of 11, 2 Bans, 6NM/3HD/3HR/4DT/3FM/1TB
+- Finals and Grand Finals
+    - Best of 13, 2 Bans, 6NM/3HD/3HR/4DT/3FM/1TB
+
+1. The referee will invite the captains of each team to the multiplayer lobby. Captains have 10 minutes to get a minimum of a 3 man roster in the lobby, otherwise the opposing team will be allowed to claim a win by default. 
+2. Both captains will `!roll`. The lowest roll will ban first, while the highest roll will ban second and pick first. 
+    - Bans and picks will alternate between teams
+3. The referee may set an !mp timer in order to limit how long teams have to make a choice. Should a team take longer than the timer, they will be forced to forfeit their choice
+    - If they run the timer on a ban, they will lose their opportunity to ban.
+    - If they run the timer on a pick, their pick will be skipped and the opposing team will be picking instead.
+4. No-Fail will be enforced on every map.
+5. Should a Freemod map be picked, at least 2 players of the team will have to choose a mod.
+    - Players are allowed to choose between Hidden, Hard Rock, Easy and Flashlight
+    - Easy scores will be multiplied by 1.75, which results in 875,000 score for an SS (excluding spinner bonus).
+6. In the case of a genuine disconnection, common sense will be applied: if proof exists of the disconnected player's score (or a rough estimate of their hypothetical score if they finished the map), it will be added to their team's total score; if it is at all ambiguous, the other team will get the point; if the disconnection occurred in the first few seconds, the map will be replayed.
+
+### Scheduling instructions
+
+#### Qualifiers
+1. Teams will be able to choose their own time, as each team will be in a separate lobby. Should a team not submit their prefered time by Friday 2 April 2021 @ 23:59 UTC+0, their country will be disqualified from the tournament.
+2. A qualifier lobby is expected to take around 90-120 minutes (1 hour and 30 minutes to 2 hours) however the pool will be designed in a way to try and reduce this.
+
+#### Round of 16 onwards
+1. Teams will be given a default schedule around Monday night on the week before their matches at the latest. Schedule will be made based on the default timezone of their country.
+    - If the default timezone causes any issues for whatever reason, captains may ask to change their country's timezone.
+2. Captains will have until Friday 23:59 UTC+0 to post any reschedules. Exceptions can be made and will be reviewed on a case by case basis.
+3. All matches will be defaulted to between Saturday 00:00 and Sunday 23:59 UTC+0. Teams may reschedule to Monday, however it would be prefered not to.
```

Comment:
specific date would be preferable here. Also might want to check date and time formatting